main.rb

```diff
@@ -1 +1,6 @@
+# frozen_string_literal: true
 
+require 'nokogiri'
+require 'open-uri'
+require 'ferrum'
+require_relative 'services/products_links/get_products_links'
```

Comment:
required/included files should be in use somehow. So add something here, what runs some code and provides some result

Reply:
so I don't think that 
```
require 'nokogiri'
require 'open-uri'
require 'ferrum'
```
are needed here